src/System.Windows.Forms/src/System/Windows/Forms/ToolStripManager.cs

```diff
@@ -1160,9 +1159,9 @@ private static ToolStripItem FindMatch(ToolStripItem source, ToolStripItemCollec
             return result;
         }
 
-        internal static ArrayList FindMergeableToolStrips(ContainerControl container)
+        internal static List<ToolStrip> FindMergeableToolStrips(ContainerControl container)
         {
-            ArrayList result = new ArrayList();
+            var result = new List<ToolStrip>();
```

Comment:
```suggestion
            List<ToolStrip> result = new();
```
Should probably be better as the type will be on the left side.